src/main/java/org/example/Country.java

```diff
@@ -0,0 +1,14 @@
+package org.example;
+
+public class Country {
+    enum Countries {
```

Comment:
We can simply declare the enum, it doesn't need a wrapper class
```suggestion
public enum Country {
```